src/popup/router/components/AccountSwitcher.vue

```diff
@@ -50,87 +101,79 @@ export default {
       return this.accounts.map((a, index) => ({ ...a, i: index })).filter((a) => a.showed);
     },
     selectedCardNumber() {
-      return this.filteredAccounts.findIndex((a) => a.i === this.activeIdx);
+      return this.filteredAccounts.findIndex((a) => a.i === this.activeAccount);
```

Comment:
This and this can be remove completly, since it is no longer used in that component.
https://github.com/aeternity/superhero-wallet/blob/fa38bb548bc23bc2760489cd9761e732bf673174/src/popup/router/components/AccountSwitcher.vue#L94-L99
https://github.com/aeternity/superhero-wallet/blob/fa38bb548bc23bc2760489cd9761e732bf673174/src/popup/router/components/AccountSwitcher.vue#L103-L105